pkg/server/server_test.go

```diff
@@ -347,6 +347,68 @@ func (m *mockStreamServer) Send(*openfgapb.StreamedListObjectsResponse) error {
 	return nil
 }
 
+// This runs TestListObjects_Unoptimized_UnhappyPaths many times over to ensure no race conditions (see https://github.com/openfga/openfga/pull/762)
+func BenchmarkListObjectsNoRaceCondition(b *testing.B) {
+	ctx := context.Background()
+	logger := logger.NewNoopLogger()
+	transport := gateway.NewNoopTransport()
+	store := ulid.Make().String()
+	modelID := ulid.Make().String()
+
+	mockController := gomock.NewController(b)
+	defer mockController.Finish()
+
+	data, err := os.ReadFile("testdata/github.json")
```

Comment:
There's not a strong reason to use the github model here given that we're only testing `repo#owner` which is just a direct relationship with an `org` type.

I vote we drop this simply in favor of

```go
typedefs := parser.MustParse(`
    type org

    type repo
        relations
            define owner: [org] as self
    `)
    
...

// use typedefs below
```

Reply:
The motivation behind this comment is because of a similar change taking place here: https://github.com/openfga/openfga/pull/763/files#diff-fe7a244e18e92d8f347af17363209172e1d79224f086b0585a1bbeef3b626c4eL360

We're using the Github model in some cases where we honestly don't need to be. Simpler is better, and in this case we can drop the dependency on the `.json` file for a simple direct relationship. Makes the test a tad cleaner and easier to manage.